packages/ra-ui-materialui/src/input/ArrayInput/SimpleFormIterator.spec.tsx

```diff
@@ -501,17 +474,14 @@ describe('<SimpleFormIterator />', () => {
                     <ArrayInput source="emails">
                         <SimpleFormIterator>
                             <FormDataConsumer>
-                                {({ getSource }) => (
+                                {() => (
```

Comment:
We don't need a FormDataConsumer then

Reply:
True but the test validates that everything works even when you have a `FormDataConsumer`